pkg/porter/publish.go

```diff
@@ -203,7 +203,7 @@ func (p *Porter) publishFromFile(ctx context.Context, opts PublishOptions) error
 
 	bundleRef.Digest, err = p.Registry.PushImage(ctx, imgRef, regOpts)
 	if err != nil {
-		return log.Errorf("unable to push CNAB invocation image %q: %w", m.Image, err)
+		return log.Errorf("unable to push CNAB bundle image %q: %w", m.Image, err)
 	}
 
 	bundleRef.Definition, err = p.rewriteBundleWithInvocationImageDigest(ctx, m, bundleRef.Digest)
```

Comment:
One second thought - it may drastically increase scope of an already large PR. I'll let you decide

Reply:
Done. I tried it and it wasn't too bad.